lib/route_model.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:core';
+import 'dart:math';
+
+import 'package:sail_routing_dart/cart_point.dart';
+
+class RouteModel {
+  Cart_Point _start;
+  Cart_Point _end;
+  List<Cart_Point> _intermediate_points;
+  double _wind_radians;
+
+  // Getters
+  Cart_Point get start => this._start;
+  Cart_Point get end => this._end;
+  List<Cart_Point> get intermediate_points => this._intermediate_points;
+  double get wind_radians => this._wind_radians;
```

Comment:
https://www.dartlang.org/guides/language/effective-dart/style I think lowerCamelCase for wind_radians